packages/paste-design-tokens/tokens/global/text-color.yml

```diff
@@ -247,23 +247,23 @@ props:
       - color-background-primary-weakest
       - color-background-destructive-weakest
   color-text-success:
-    value: "{!palette-green-60}"
+    value: "{!palette-green-70}"
     comment: Text color for success text.
     uicontrol_contrast_pairing:
       # - color-background
       # - color-background-body
       - color-background-body-inverse
       # - color-background-row-striped
   color-text-warning:
-    value: "{!palette-orange-60}"
+    value: "{!palette-orange-70}"
     comment: Color for warning text.
     uicontrol_contrast_pairing:
       # - color-background
       # - color-background-body
       - color-background-body-inverse
       # - color-background-row-striped
   color-text-warning-strong:
-    value: "{!palette-orange-70}"
+    value: "{!palette-orange-80}"
```

Comment:
With these increased color contrast ratios are there any of these contrast pairings that now work and can be uncommented?

Reply:
> With these increased color contrast ratios are there any of these contrast pairings that now work and can be uncommented?

Uncommented and tests are passing 👍 

> I'm also assuming the these are truly text color tokens now and should also be having text contrast pairings

Trying to add these in but every text contrast pairing test is failing with the new tokens and I'm not sure why